static/css/devhub/forms.less

```diff
@@ -19,6 +19,9 @@
         tr.version-disabled {
             opacity: 0.5;
         }
+        td {
+            max-width: 547px;
```

Comment:
@shatgupt does this work without adding a max-width?

Reply:
It works without max-width if there is no long unbroken string but if it is there, it overflows and gets hidden.

Without long unbroken string:
![screenshot 17](https://user-images.githubusercontent.com/2388644/46558017-9da5a000-c8a0-11e8-9099-add8c494846b.png)

With long unbroken string:
![screenshot 16](https://user-images.githubusercontent.com/2388644/46558006-99798280-c8a0-11e8-982d-5d36740fa3d1.png)

Reply:
Ah ok, thanks for checking.